operate/common/src/main/java/io/camunda/operate/property/OperateProperties.java

```diff
@@ -374,4 +375,20 @@ public boolean isRfc3339ApiDateFormat() {
   public void setRfc3339ApiDateFormat(boolean rfc3339ApiDateFormat) {
     this.rfc3339ApiDateFormat = rfc3339ApiDateFormat;
   }
+
+  public String getIndexPrefix() {
```

Comment:
At some point in the future, we may want to create a wrapper component that holds the Elastic/Opensearch properties and makes the call to DatabaseInfo. This would allow checks for db type to be contained in one place and all client components just make an abstract property get call.